### PR TITLE
refactor vehicleGen to be more Pythonic

### PR DESCRIPTION
**Pull request by behzadmehrabi**

### CODE REVIEW

File: vehicleGen.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def car_plate():
 
     compute = csalp2 + csalp3 + csnum1 + csnum2 + csnum3 + csnum4
 
-    number = str(gen_num1) + str(gen_num2) + str(gen_num3) + str(gen_num4)
+    number = str(gen_num1 + gen_num2 + gen_num3 + gen_num4)
```
Comment (MythreyaK) [inline vehicleGen.py]: I think that's incorrect. What you're doing is adding up the numbers and returning that. The `str` for each of them is required to get a string (ex: `3456`; which is what I think is required).

File: vehicleGen.py [hunk 2]
```diff
@@ -154,7 +154,7 @@ def goods_plate():
...
```
Comment (MythreyaK) [inline vehicleGen.py]: Same here

**Review by MythreyaK — CHANGES_REQUESTED** [review]
Please have a look at the comments

**Comment by Stewart86** [comment]
thank you!

**Pull request merged**